requirements_docs.txt

```
@@ -1,6 +1,7 @@
 Sphinx==4.5.0
 cmocean==2.0
 colorcet==3.0.0
+enum_tools
```

Just a drive-by comment:
1. Do we really need to add `enum_tools` and `sphinx-toolbox`? (Asking just because we should have very good reasons for adding additional dependencies, even if just for doc building.)
2. We should pin these dependencies too and let dependabot bump the versions as they are released.

---

I don't see where `sphinx-toolbox` is used. I'm fine with adding `enum_tools` provided we pin the dependency.

---

`sphinx-toolbox` appears to be an unlisted dependency of `enum_tools` (the docs build failed without it). I can pin the version numbers, though.

---

Now it seems v0.9.0 (What is 0.9.0.post1?).
```suggestion
enum-tools==0.9.0
```